src/main/java/recipeio/commands/FindCommand.java

```diff
@@ -0,0 +1,84 @@
+package recipeio.commands;
+
+import recipeio.recipe.Recipe;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class FindCommand {
+    public static void findKeyword(String keyword, ArrayList<Recipe> recipes) {
+        ArrayList<Recipe> matches = new ArrayList<>();
+        if (recipes.isEmpty()) {
+            System.out.println("Sorry, you have no recipes to find matches with. Try adding some!");
+        } else {
+            for (Recipe recipe : recipes) {
+                if (recipe.name.contains(keyword)) {
+                    matches.add(recipe);
+                }
+            }
+            if (matches.isEmpty()) {
+                System.out.println("There were no matches. Try searching for something else.");
+            } else {
+                int counter = 0;
+                StringBuilder output = new StringBuilder();
+                while (counter < matches.size()) {
+                    String recipeNumber = Integer.toString(counter + 1);
+                    Recipe selected = matches.get(counter);
+                    output.append(selected.toString());
+                    if (counter != matches.size() - 1) {
+                        output.append("\n");
+                    }
+                    counter += 1;
+                }
+                System.out.println("Here are your matches:\n" + output);
```

Comment:
Lambda expressions may help to reduce length of code here too.

You can consider the .filter() method.

You may also extract the method that simply prints the matches. I'm not too familiar with StringBuilder, but I think you can achieve something similar by looping or using forEach (lambda expression).